src/components/Navbar.js

```diff
@@ -1,28 +1,26 @@
 import { NavLink } from 'react-router-dom';
 import { IoMdContact } from 'react-icons/io';
-import '../App.css';
+import styles from '../styles/Navbar.module.css';
 
 const links = [
   { path: '/', text: 'Books', exact: true },
   { path: 'Categories', text: 'Categories' },
 ];
 const Navbar = () => (
-  <nav className="navbar">
-    <ul className="links">
-      <li>
-        <span className="title">Bookstore CMS</span>
-      </li>
-      {links.map((link) => (
-        <li key={link.text}>
-          <NavLink to={link.path} className="navLink" exact={link.exact}>
-            {link.text}
-          </NavLink>
-        </li>
-      ))}
-    </ul>
-    <div className="icons">
-      <IoMdContact title="Contact" />
+  <nav className={styles.navbar}>
+    <div className={styles.links}>
+      <span className={styles.title}>Bookstore CMS</span>
+      <ul className={styles.navList}>
+        {links.map((link) => (
+          <li key={link.text}>
+            <NavLink to={link.path} className={styles.navLink}>
+              {link.text}
+            </NavLink>
+          </li>
+        ))}
+      </ul>
     </div>
+    <IoMdContact className={styles.icon} />
```

Comment:
[REQUIRED] Kindly ensure the user icon looks just like the one in the template. This trains users to stick customer requirements

![image](https://github.com/AbdelazizSoliman/bookstore/assets/62938735/10014d71-81e0-4f7c-80b8-ad6450dd6ebf)